core/domain/study_guide_domain.py

```
@@ -354,9 +354,8 @@ def to_subtopic_page_dict_for_android(self) -> StudyGuideAndroidDict:
         for section in self.sections:
             section_dict = section.to_dict()
             heading_html = (
-                '<p><strong>'
-                + f'{section_dict["heading"]["unicode_str"]}'
-                + '</strong></p>'
+                '<p><strong>%s</strong></p>'
+                % section_dict['heading']['unicode_str']
```

Ruff (0.15.21)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)